ReviewerUI/webpack.config.js

```diff
@@ -0,0 +1,31 @@
+var debug = process.env.NODE_ENV !== "production";
```

Comment:
another thing, as a good practice avoid using vars if you dont need changes. Use `let` and `const`